diff --git a/src/util/StreamLink.cpp b/src/util/StreamLink.cpp
index 379c2699..9b75d2d5 100644
--- a/src/util/StreamLink.cpp
+++ b/src/util/StreamLink.cpp
@@ -209,7 +209,7 @@ void openStreamlink(const QString &channelURL, const QString &quality,

 void openStreamlinkForChannel(const QString &channel)
 {
-    static const QString INFO_TEMPLATE("Opening %1 in Streamlink ...");
+    static const QString INFO_TEMPLATE("Opening %1 in twitch-hls-client ...");

     auto *currentPage = dynamic_cast<SplitContainer *>(
         getApp()->windows->getMainWindow().getNotebook().getSelectedPage());
diff --git a/src/widgets/helper/CommonTexts.hpp b/src/widgets/helper/CommonTexts.hpp
index 83a2447f..8b483613 100644
--- a/src/widgets/helper/CommonTexts.hpp
+++ b/src/widgets/helper/CommonTexts.hpp
@@ -3,6 +3,6 @@
 #define OPEN_IN_BROWSER "Open stream in browser"
 #define OPEN_PLAYER_IN_BROWSER "Open player in browser"
 #define OPEN_MOD_VIEW_IN_BROWSER "Open mod view in browser"
-#define OPEN_IN_STREAMLINK "Open in streamlink"
+#define OPEN_IN_STREAMLINK "Open in twitch-hls-client"
 #define DONT_OPEN "Don't open"
 #define OPEN_WHISPERS_IN_BROWSER "Open whispers in browser"
